docs/src/howto-guides/018_update_config_paths.md

```diff
@@ -0,0 +1,58 @@
+# How to change default paths
+
+The `tedge config set` command can be used to change various file system paths used by tedge components.
+The following table captures the paths that can be changed, with their default locations.
+
+| Config Key | Description | Default Value |
+|------------|-------------|---------------|
+| tmp.path | Directory where temporary files are created/stored. E.g: while downloading files | `/tmp` |
+| logs.path | Directory where log files are created | `/var/log` |
+| run.path | Directory where runtime information are stored | `/run` |
+| data.path | Directory where all persistent data files are stored. E.g: Cached binary files, operation metadata etc | `/var/tedge` |
```

Comment:
I am not sure **persistent** is a right word. `/var` directory is the directory to be written during a process. http://www.linfo.org/var.html